front/app/containers/Admin/messaging/AutomatedEmails/ExampleModal/ExampleFrame.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// Components
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import Frame from 'react-frame-component';
+
+// Hooks
+import { ICampaign } from 'services/campaigns';
+import { ICampaignExampleData } from 'api/campaign_examples/types';
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+
+// i18n
+import T from 'components/T';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+
+const StyledFrame = styled(Frame)`
+  border-radius: ${(props) => props.theme.borderRadius};
+  border: 1px solid ${(props) => props.theme.colors.divider};
+  width: 100%;
+  height: 450px;
+`;
```

Comment:
I'm not rendering a div, vut a `Frame`. Don't think there is a way around it, or that we need a way around it

Reply:
Yes, I meant you can wrap with a Box and put the props on there, but no need to change.

Reply:
Don't think that will work, since the Frame has default styling that needs to be overridden, it's not the same than styling a wrapping div. Anyway, won't change ;)